specs/phase1/beacon-chain.md

```diff
@@ -188,7 +289,9 @@ class Validator(Container):
     # (of the particular validator) in which the validator is activated
     # = get_custody_period_for_validator(...)
     next_custody_secret_to_reveal: uint64
-    max_reveal_lateness: Epoch
+    # TODO: The max_reveal_lateness doesn't really make sense anymore.
```

Comment:
We will need a new way to incentivise people to reveal their custody secrets timely. Without the bit challenges, the max_reveal_lateness doesn't really make sense anymore.

Reply:
@vbuterin @djrtwo

Reply:
Created issue #1888